frontend/yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@7.5.5", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.5.5":
   version "7.5.5"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.5.5.tgz#bc0782f6d69f7b7d49531219699b988f669a8f9d"
-  integrity sha512-27d4lZoomVyo51VegxI20xZPuSHusqbQag/ztrBC7wegWoQ1nLREPVSKSW8byhTlzTKyNE4ifaTA6lCp7JjpFw==
```

Comment:
Are these changes due to different yarn versions that we are using? What version do you have?

Reply:
I have yarn 1.5.0, will revert this change and check if works well.

Reply:
yarn 1.5.0 is almost 2 years old. Could you update to latest release and rebuild?

Reply:
Updated, Have an issue with wrong yarn version used. By default was using 1.5.0 instead of 1.21.1